plotly/plotly/plotly.py

```diff
@@ -1216,6 +1218,31 @@ def validate_credentials(credentials):
         raise exceptions.PlotlyLocalCredentialsError()
 
 
+def secret_key(r, **payload):
+    """
+    update plot's url to include the scret key
```

Comment:
:dolls: ~~`update`~~ `Update` and document arguments please :smile_cat: 

Can we also pick a better variable name than `r`?